main/main.cpp

```diff
@@ -121,7 +121,7 @@ static int audio_driver_idx = -1;
 
 // Engine config/tools
 
-static bool single_window = false;
+static bool single_window = true;
```

Comment:
I'm guessing this is to restrict the editor to be within a single window since Godot 4.0 will allow for multiple windows. I thought they were a programmatic way to control that settings?

Reply:
I changed this manualy for testing purposes, yes, I want it to be automatic by testing multiwondow support before but I didn't had the time, now that the code is clearer and everything seems to work Ill start to clean up things including this

Reply:
fixed, can you check? I want to cherrypick that to a new PR